### ISSUE TITLE
Improve build process

### ISSUE THREAD

**Issue by alwaysblank** [alwaysblank/elfin#1]
The primary reason for the current build process is that I want to have my build process generate separate JS files for modern and legacy browsers, instead of serving a huge polyfilled mess to everyone. I'm using the technique described here: https://philipwalton.com/articles/deploying-es2015-code-in-production-today/ In fact, I'm using an only slightly modified version of his proof-of-concept example. While it functions just fine, there are a couple things I'd like to do that I think could improve it:

- [ ] Run asset builds in parallel. AFAIK there's no reason they can't, and in theory it should improve build performance (esp. if I could get them to run on separate cores...maybe spawning new Node processes...?).
- [ ] Provide more useful user feedback during the build process.
- [ ] Use webpack's dev server (or a similar solution) to serve the site during development and provide HMR. Currently it just rebuilds the entire thing whenever a file changes, which isn't terribly slow, but could be much faster.

**Comment by alwaysblank** [comment]
These are no longer relevant
